signer/src/storage.rs

```diff
@@ -82,6 +82,13 @@ pub trait DbRead {
         &self,
         block_id: StacksBlockId,
     ) -> impl Future<Output = Result<bool, Self::Error>> + Send;
+
+    /// Return the applicable DKG shares for the
+    /// given aggregate key
+    fn get_encrypted_dkg_shares(
+        &self,
+        aggregate_key: &model::PubKey,
+    ) -> impl Future<Output = Result<Option<model::EncryptedDkgShares>, Self::Error>> + Send;
```

Comment:
Shouldn't the output be `Result<Vec<model::EncryptedDkgShares>, Self::Error>>`, since you get an `EncryptedDkgShares` per signer peer? If not, maybe its `encrypted_shares` field should be a `Vec<Bytes>`?

Reply:
After looking at the `store_dkg_shares` function, it seems like this is only supposed to contain the signers private share, not any of the secret shares shared with it during DKG. Is that right?

Reply:
After looking yet again, this is the right signature. No need to change anything.

Reply:
Yeah exactly, we shouldn't hold on to any additional private information. You'll only get a single `EncryptedDkgShares` per signer, and that object would hold multiple shares if the party has multiple shares (which in `v1` it won't afaik).